src/components/NFTBook/TippingDialog.vue

```diff
@@ -50,16 +58,54 @@
               <Label class="text-dark-gray" preset="h4">{{ tipping }}</Label>
               <Label preset="h6" class="text-medium-gray">{{ currency }}</Label>
             </div>
-          </ButtonV2>
+            <div
+              v-if="selectedIndex === i && !inputValue"
+              :class="[
+                'w-[20px]',
+                'h-[20px]',
+                'flex',
+                'justify-center',
+                'items-center',
+                'rounded-[50%]',
+                'bg-like-green',
+                'border-[1px]',
+                'border-white',
+                'absolute',
+                'top-[-10px]',
+                'left-[-10px]',
+              ]"
+            >
+              <IconCheckMini class="text-white" />
+            </div>
+          </div>
         </div>
-        <div class="flex justify-center w-full mt-[8px]">
+        <div class="flex justify-center w-full mt-[8px] relative">
           <NFTBookTippingInput
             v-model="inputValue"
             :fixed-text="currency"
             :placeholder="$t('nft_tipping_placeholder')"
             @input="handleInputChange"
             @on-focus="selectedIndex = -1"
           />
+          <div
+            v-if="selectedIndex === -1"
+            :class="[
+              'w-[20px]',
+              'h-[20px]',
+              'flex',
+              'justify-center',
+              'items-center',
+              'rounded-[50%]',
+              'bg-like-green',
+              'border-[1px]',
+              'border-white',
+              'absolute',
+              'top-[-10px]',
+              'left-[-10px]',
+            ]"
+          >
+            <IconCheckMini class="text-white" />
+          </div>
```

Comment:
Seems can be refactorized

Reply:
[♻️ Refactor check icon](https://github.com/likecoin/liker-land/pull/1621/commits/1bf6a7014400f7febed25c82b71fa09ce31bc088)